pages/linux/bluetoothd.md

```diff
@@ -0,0 +1,24 @@
+# bluetoothd
+
+> Daemon to manage bluetooth devices.
+> More information: <https://manned.org/bluetoothd>.
+
+- Start the daemon:
+
+`bluetoothd`
+
+- Start the daemon, logging to stdout:
+
+`bluetoothd --nodetach`
+
+- Start the daemon with a specific configuration file (defaults to `/etc/bluetooth/main.conf`):
+
+`bluetoothd --configfile {{path/to/file}}`
+
+- Start the daemon with verbose output to stderr:
+
+`bluetoothd --debug`
+
+- Start the daemon with verbose output for a specific file:
+
+`bluetoothd --debug={{path/to/file}}`
```

Comment:
And maybe this?
```suggestion
`bluetoothd --debug={{path/to/file1}}:{{path/to/file2}}:{{path/to/file3}}`
```

Reply:
How we are specifying this now? @marchersimon @CleanMachine1

Reply:
This suggestion is fine @navarroaxel - as I mentioned in the issue we discussed this it's usually a case-by-case basis.